opal/mca/btl/usnic/btl_usnic_component.c

```diff
@@ -630,7 +630,8 @@ static mca_btl_base_module_t** usnic_component_init(int* num_btl_modules,
     hints.ep_attr = &ep_attr;
     hints.fabric_attr = &fabric_attr;
 
-    ret = fi_getinfo(FI_VERSION(1, 0), NULL, 0, 0, &hints, &info_list);
+    ret = fi_getinfo(FI_VERSION(FI_MAJOR_VERSION, FI_MINOR_VERSION),
```

Comment:
No, that would make a run-time query, as opposed to a compile-time query.  The way it is written, it will tell libfabric "this is the version of the API with which I was compiled."